internal/system/agent/executor/compose.go

```diff
@@ -20,7 +20,7 @@ func WasDockerContainerComposeStarted(service string) bool {
 	cmdName := "docker"
 	cmdArgs := []string{"ps"}
 	if cmdOut, err = exec.Command(cmdName, cmdArgs...).Output(); err != nil {
-		fmt.Fprintln(os.Stderr, "There was an error running the docker-compose command: ", err)
+		fmt.Fprintln(os.Stderr, "There was an error running the docker-compose command: ", err.Error())
```

Comment:
MIssing `%s` in first string. Nothing to replace with err.Error()

Reply:
Also should be consistent between usage of `fmt.Fprintln` and `log.Printf`. Either use one or the other, otherwise logs look really confusing with some lines having the log prefix and some not having the prefix.

Reply:
> Also should be consistent between usage of `fmt.Fprintln` and `log.Printf`. Either use one or the other, otherwise logs look really confusing with some lines having the log prefix and some not having the prefix.

- We got [Issue #780](https://github.com/edgexfoundry/edgex-go/issues/780) open for that.